tirehtoori/urls.py

```diff
@@ -37,3 +41,25 @@
         urlpatterns.append(re_path(rf"^(?!{settings.ADMIN_URL})", api.urls))
     else:
         urlpatterns.append(path("", api.urls))
+
+
+#
+# Kubernetes liveness & readiness probes
+#
+@require_GET
+def healthz(*args, **kwargs):
+    return HttpResponse(status=200)
+
+
+@require_GET
+def readiness(*args, **kwargs):
+    response_json = {
+        "status": "ok",
+        "packageVersion": __version__,
+        "commitHash": settings.COMMIT_HASH,
+        "buildTime": settings.APP_BUILD_TIME.strftime("%Y-%m-%dT%H:%M:%S.000Z"),
+    }
+    return JsonResponse(response_json, status=200)
+
+
+urlpatterns = [path("healthz", healthz), path("readiness", readiness)] + urlpatterns
```

Comment:
I wonder if these should be `__healthz` and `__readiness`? :thinking:

Reply:
Yeah, that's probably better. The idea behind static being `__static` is to avoid potential conflicts with redirects,, i.e. a path is reserved for non-redirect uses. A redirect rule with a static path (like `https://acme.test/static`) is rather likely while `https://acme.test/__static` is probably not very likely.